src/main/java/org/asciidoc/intellij/lexer/asciidoc.flex

```diff
@@ -219,14 +219,14 @@ ANCHOREND = "]"
 }
 
 <ATTRIBUTE_DECL> {
-  "\n"                 { yybegin(MULTILINE); return AsciiDocTokenTypes.LINE_BREAK; }
+  "\n"                 { yybegin(YYINITIAL); return AsciiDocTokenTypes.LINE_BREAK; }
```

Comment:
@bbrenne - i think it is safe to return YYINITIAL as no state is kept here, same below